pkgs/by-name/um/umu-launcher-unwrapped/package.nix

```
@@ -0,0 +1,67 @@
+{
+  python3Packages,
+  fetchFromGitHub,
+  lib,
+  bash,
+  hatch,
+  scdoc,
+  replaceVars,
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "umu-launcher-unwrapped";
+  version = "1.1.4";
+
+  src = fetchFromGitHub {
+    owner = "Open-Wine-Components";
+    repo = "umu-launcher";
+    tag = version;
+    hash = "sha256-TOsVK6o2V8D7CLzVOkLs8AClrZmlVQTfeii32ZIQCu4=";
+  };
+
+  patches = [
+    # This should fail, and be removed for releases after 1.1.4,
+    # see https://github.com/Open-Wine-Components/umu-launcher/pull/289
+    (replaceVars ./no-umu-version-json.patch { inherit version; })
+    # Patch the python installer call to use --prefix instead of --destdir
+    # Ideally, a change should be upstreamed, that adds both arguments
+    # if their respective variables are set.
```

```suggestion
    # See https://github.com/Open-Wine-Components/umu-launcher/pull/343
```

I've opened an upstream PR, so let's link to it :upside_down_face:

---

```diff
diff --git a/pkgs/by-name/um/umu-launcher-unwrapped/makefile-installer-prefix.patch b/pkgs/by-name/um/umu-launcher-unwrapped/makefile-installer-prefix.patch
deleted file mode 100644
index 890d6537e33c..000000000000
--- a/pkgs/by-name/um/umu-launcher-unwrapped/makefile-installer-prefix.patch
+++ /dev/null
@@ -1,13 +0,0 @@
-diff --git a/Makefile.in b/Makefile.in
-index b82053d..3e4379e 100644
---- a/Makefile.in
-+++ b/Makefile.in
-@@ -90,7 +90,7 @@ umu-dist: $(OBJDIR)/.build-umu-dist
- umu-dist-install: umu-dist
- 	$(info :: Installing umu )
- 	install -d $(DESTDIR)$(PYTHONDIR)/$(INSTALLDIR)
--	$(PYTHON_INTERPRETER)  -m installer --destdir=$(DESTDIR) $(OBJDIR)/*.whl
-+	$(PYTHON_INTERPRETER) -m installer --prefix=$(PREFIX) $(OBJDIR)/*.whl
- 
- ifeq ($(FLATPAK), xtrue)
- umu-install: version-install umu-dist-install
diff --git a/pkgs/by-name/um/umu-launcher-unwrapped/package.nix b/pkgs/by-name/um/umu-launcher-unwrapped/package.nix
index 357ef7a5d806..1f97d0c31580 100644
--- a/pkgs/by-name/um/umu-launcher-unwrapped/package.nix
+++ b/pkgs/by-name/um/umu-launcher-unwrapped/package.nix
@@ -1,6 +1,7 @@
 {
   python3Packages,
   fetchFromGitHub,
+  fetchpatch2,
   lib,
   bash,
   hatch,
@@ -22,10 +23,12 @@ python3Packages.buildPythonPackage rec {
     # This should fail, and be removed for releases after 1.1.4,
     # see https://github.com/Open-Wine-Components/umu-launcher/pull/289
     (replaceVars ./no-umu-version-json.patch { inherit version; })
-    # Patch the python installer call to use --prefix instead of --destdir
-    # Ideally, a change should be upstreamed, that adds both arguments
-    # if their respective variables are set.
-    ./makefile-installer-prefix.patch
+    # Patch the python installer call to support PREFIX
+    # See https://github.com/Open-Wine-Components/umu-launcher/pull/343
+    (fetchpatch2 {
+      url = "https://github.com/Open-Wine-Components/umu-launcher/pull/343.diff";
+      hash = "sha256-+h0ao2WUVUUR/AHEtGj5IAIUVivfAc+asnr0632hN74=";
+    })
   ];
 
   nativeBuildInputs = [
```